inflammation-analysis.py

```diff
@@ -21,7 +21,7 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data)}
+        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), **(models.s_dev(inflammation_data))}
```

Comment:
Maybe I do not understand enough about python, but is leaving a **(models....) a good idea? Why not use a 'std_dev' key?

Reply:
Thanks for the input, I might not understand this function well enough myself :) Could you elaborate a bit more or maybe even add a suggestion? Much appreciated.